### PR TITLE
Update refined to 0.9.3

### PR DESCRIPTION
**Pull request by fthomas**

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -370,7 +370,7 @@ lazy val refinedBase = circeCrossModule("refined", mima = previousCirceVersion)
   .settings(
     libraryDependencies ++= Seq(
       "eu.timepit" %%% "refined" % refinedVersion,
-      "eu.timepit" %%% "refined-scalacheck" % refinedVersion % Test
+      "eu.timepit" %%% "refined-scalacheck_1.13" % refinedVersion % Test
```
Comment (fthomas) [inline build.sbt]: This has been changed because `refined-scalacheck` depends on ScalaCheck 1.14 but circe is still on 1.13.

**Comment by fthomas** [comment]
The 2.13.0-M4 build failed because refined 0.9.3 is only available for 2.13.0-M5.

**Comment by travisbrown** [comment]
@fthomas Yes, we're still waiting on Jawn. I'll try to get this sorted this week.

**Comment by travisbrown** [comment]
@fthomas Thanks again! I've included this commit in #1015.

**Pull request closed without merging**